src/simulate/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# this is getting the latest under 3.6 version
+FROM docker.io/python:3
+# Setting the default directory in container
+WORKDIR /usr/src/app
+# copies requirements.txt from the build source directory into the container
+COPY requirements.txt ./
```

Comment:
As in my PR comments to @dlhaines about putting kartograafr in Docker, maybe it's better to remove `WORKDIR` and be more explicit with the destination argument for `COPY`.